lta/bundler.py

```diff
@@ -233,6 +235,21 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
         }
         await lta_rc.request('POST', '/Files/actions/bulk_update', update_body)
 
+
+def check_drain_semaphore() -> bool:
+    """Check if a drain semaphore exists in the current working directory."""
+    cwd = os.getcwd()
+    drain_filename = os.path.join(cwd, DRAIN_SEMAPHORE_FILENAME)
+    return Path(drain_filename).exists()
```

Comment:
This function seems like duplicate code, since it's in every module.

Reply:
This is one that will probably end up refactored in DEV-605:
https://wipacit.atlassian.net/browse/DEV-605